apps/web/src/utils/url.ts

```diff
@@ -43,3 +43,14 @@ export const getOriginPath = (url: string): string => {
     return url
   }
 }
+
+// Function to strip URL parameters, returning only the base URL
+export const stripURLParams = (url: string): string => {
```

Comment:
To match the naming of other functions in the codebase.

```suggestion
export const stripUrlParams = (url: string): string => {
```